### ISSUE TITLE
Unhandled promise rejections

### ISSUE THREAD

**Issue by c3s4r** [devsu/condor-framework#34]
When making a wrong call, condor responds with error (which is correct), but in the server, this warning is logged:

```
(node:5672) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): Error: startBatch failed
(node:5672) DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

We need to make sure that promise rejections are properly handled.

**Comment by c3s4r** [comment]
Partially fixed on 1.2.1

**Comment by c3s4r** [comment]
@jdgutierrez: This is still happing when condor launches the following error: `'No response send after error was handled'`. Please take a look at it.

**Comment by c3s4r** [comment]
Fixed in `1.3.0`
